redfish-core/lib/assembly.hpp

```diff
@@ -407,15 +560,83 @@ inline void setAssemblyLocationIndicators(
         {
             setLocationIndicatorActive(asyncResp, assembly, iter->second);
         }
+
+        auto iter2 = oemIndicatorMap.find(std::to_string(assemblyIndex));
+
+        if (iter2 != oemIndicatorMap.end())
+        {
+            std::optional<nlohmann::json> openbmc;
+            if (!json_util::readJson(iter2->second, asyncResp->res, "OpenBMC",
+                                     openbmc))
+            {
+                BMCWEB_LOG_ERROR("Property Value Format Error ");
+                messages::propertyValueFormatError(
+                    asyncResp->res,
+                    (*openbmc).dump(2, ' ', true,
+                                    nlohmann::json::error_handler_t::replace),
+                    "OpenBMC");
+                return;
+            }
+
+            if (!openbmc)
+            {
+                BMCWEB_LOG_ERROR("Property Missing ");
```

Comment:
Not an error here since user error, DEBUG here

Reply:
Fixed